src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1115,6 +1121,13 @@ def get_cls(name, info, parent=None, version=None):
             bases = bases + (_CreatableNamedObjectMixin,)
         elif obj_type == "named-object":
             bases = bases + (_NonCreatableNamedObjectMixin,)
+        elif info.get("has-allowed-values", False) and version in [
+            "222",
+            "231",
+            "232",
+            "241",
```

Comment:
Sorry, I am totally confused now. Do we need to add every future release in this list from now on?

Reply:
We tried this earlier, `elif info.get("has-allowed-values") and version in ["222", "231", "232", "241"]:` but this is also not working. It is failing again.

Reply:
Do we need to add every future release in this list from now on? - Not sure, yet. It is not working for now.

Reply:
@h-krishnan @seanpearsonuk 

We have observed following scenarios - 

1. `elif info.get("has-allowed-values") or version == "222":` 

`Build` check fails.

`--------------------------------------------------------------`

2. `elif info.get("has-allowed-values") and version in ["222", "231", "232", "241"]:` 

`Build` check passes but test fails in all versions.

![image](https://github.com/ansys/pyfluent/assets/106588300/6d22dec3-3cd4-4aba-bfa2-6f5532d42ea4)

`--------------------------------------------------------------`

2. `elif info.get("has-allowed-values", False) and version in ["222", "231", "232", "241"]:` 

`Build` check passes but test fails in all versions.

![image](https://github.com/ansys/pyfluent/assets/106588300/4ec3cbaf-f45b-49c1-96a2-b1bc6599953a)

`--------------------------------------------------------------`

Reply:
`elif name in _items_with_string_and_allowed_values:` is working fine for all versions.